util/src/test/java/tech/pegasys/artemis/util/mikuli/G1PointTest.java

```diff
@@ -214,4 +214,27 @@ void succeedsWhenAttemptToDeserialiseWithBFlagAndAFlagTrueThrowsIllegalArgumentE
         IllegalArgumentException.class,
         () -> G1Point.fromBytesCompressed(Bytes.fromHexString(xInput)));
   }
+
+  @Test
+  void succeedsWhenDifferentPointsHaveDifferentHashcodes() {
+    G1Point point1 = G1Point.random();
+    G1Point point2 = G1Point.random();
```

Comment:
We should use a seed for the random generation so we get consistent points generated.  hashCode will sometimes collide so the test would be intermittent when using unseeded random values.  Then the check for equals can be skipped.

Reply:
OK.